internal/cmd/backup.go

```diff
@@ -78,7 +147,17 @@ func backupCmdFunc(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("`backup` is not supported on this version of SpiceDB")
 	}
 
-	encoder, err := backupformat.NewEncoder(relWriter, schemaResp.SchemaText, schemaResp.ReadAt)
+	// Skip any definitions without the provided prefix
+	schema := schemaResp.SchemaText
+	prefixFilter := cobrautil.MustGetString(cmd, "prefix-filter")
+	if prefixFilter != "" {
+		schema, err = filterSchemaDefs(schema, prefixFilter)
```

Comment:
There is the opportunity here to provide a prefix of `fire`, expecting to get only `fire/` but also getting `firetruck/` on accident. Do we want to add any logic around the separators?

Reply:
I can change it so that it's everything to the left of the final `/` just to be 1000% sure

Reply:
✅